packages/workflow/src/internals.ts

```diff
@@ -695,8 +696,12 @@ export class Activator implements ActivationHandler {
   }
 }
 
+export function maybeGetActivator(): Activator | undefined {
+  return (globalThis as any).__TEMPORAL_ACTIVATOR__;
+}
```

Comment:
This can be refactored to avoid referencing `__TEMPORAL_ACTIVATOR__` in some of the other places.